docs/rules/no-interpolation-in-snapshots.md

```diff
@@ -0,0 +1,60 @@
+# Disallow string interpolation inside inline snapshots (`no-interpolation-in-snapshots`)
```

Comment:
```suggestion
# Disallow string interpolation inside snapshots (`no-interpolation-in-snapshots`)
```

Might make sense to drop the `inline` given the name change, just to keep things short and snappy 🤔